src/Pages/Applications/DevtronApps/Details/AppConfigurations/MainContent/CompareConfigView.tsx

```diff
@@ -19,8 +19,8 @@ const CompareConfigView = ({
     isDeleteOverrideView,
     editorKey = `${compareFromSelectedOptionValue || 'compare'}-draft-editor-key`,
 }: CompareConfigViewProps) => (
-    <div className="flexbox-col dc__overflow-scroll">
-        <div className="dc__grid-half">
+    <div className="flexbox-col">
+        <div className="dc__grid-half bcn-0 dc__position-sticky dc__top-0 dc__zi-10">
```

Comment:
Can we add a less z-index?

Reply:
CodeEditor scrollbar has 9 z-index so 10 is our best option